index.bs

```diff
@@ -749,19 +749,23 @@ the result of [=create a frozen array|creating a frozen array=] from running [=c
 When asked to <dfn>create brands</dfn> with |version type|, run the following steps:
 
 1. Let |list| be a [=/list=].
-1. Collect pairs of [=user agent/brand=] and versions of type |version type| which represent the
-    [=user agent=] or [=equivalence classes=].
-1. For each pair:
-    1. Let |dict| be a new {{NavigatorUABrandVersion}} dictionary, with [=user agent/brand=] as
-        {{NavigatorUABrandVersion/brand}} and a string of type |version type| as
-        {{NavigatorUABrandVersion/version}}.
+1. For each [=user agent/brand=] that represents the [=user agent=]—or an [=equivalence class=]—as
+    |brand|:
+    1. Let |version| be a [=/string=].
+    1. If |version type| is "full version", set |version| to a string that corresponds to the
```

Comment:
Assert it's either one or the other

Reply:
Nice, didn't realize https://infra.spec.whatwg.org/#assertions existed.